DependencyInjection/Configuration.php

```diff
@@ -225,4 +227,25 @@ private function addContentListenerSection(NodeBuilder $nodeBuilder)
             ->end()
         ;
     }
+
+    /**
+     * Attach the form node to the tree.
+     *
+     * @param NodeBuilder $nodeBuilder
+     */
+    private function addFormSection($nodeBuilder)
+    {
+        $nodeBuilder
+           ->arrayNode('form')
+                ->canBeDisabled()
+                ->children()
+                    ->arrayNode('data_class')
```

Comment:
afaik symfony fixes that automatically, when we use _ in our config that
you can use - in xml.